framework/source/class/qx/ui/table/cellrenderer/Abstract.js

```diff
@@ -168,7 +168,9 @@ qx.Class.define("qx.ui.table.cellrenderer.Abstract",
      */
     _getCellAttributes : function(cellInfo)
     {
-      return "";
+      var cellId = "qooxdoo-table-cell-" + cellInfo.table.toHashCode() + "-" + cellInfo.row + "-" + cellInfo.col;
+      var readOnly = cellInfo.editable !== null && cellInfo.editable !== undefined ? !cellInfo.editable : true;
+      return "id=" + cellId + " role=gridcell aria-readonly=" + readOnly;
```

Comment:
This is likely ok, but @cboulanger please ensure this `id` doesn't conflict with the ones your id stuff for testing imposes. (Or are those called `qxId` or some such thing?)

Reply:
Could you change id to some other name?

Reply:
The qooxdoo id system doesn't use DOM ids, see https://qooxdoo.org/documentation/6.0/#/core/object_id?id=finding-objects-via-the-dom - so no conflict here. I think it is safe to use the id as done here since it contains a hash that is very unlikely to conflict with any other id. At the same time, the implementation could of course use the qooxdoo id system, so each cell could be looked up by `table.getObject(id)`. It depends on what @ArturWesner needs the ids for - I didn't have a close look at the code.